### ISSUE TITLE
Nourishment

### ISSUE THREAD

**Issue by smudge202** [smudge202/clean-living#1]
- We need the ability to track our level of hunger.
- We need a representation of Hunger as a percentage.
- Hunger should diminish over time.
- Hunger should be replenished by eating.
- The player should be adversely affected by starvation
- The player should have positive effects when well nourished

**Comment by smudge202** [comment]
This task is available now that engine-time has collapsed to dev.

**Comment by smudge202** [comment]
I think this should be blocked pending #11

**Comment by herecydev** [comment]
Certainly things will change with #11 , but shouldnt be too much to change on this end.

Done:
We need the ability to track our level of hunger
Hunger should diminish over time

Will not do:
The player should be adversely affected by starvation
The player should have positive effects when well nourished

(These will be the responsibility of some wellness/health component)

**Comment by herecydev** [comment]
Changed to Nourishment. So much confusion over increase in hunger =/= positive.

**Comment by herecydev** [comment]
Done:
Hunger should be replenished by eating - this is done in a simple, instant patch up way. It might be that it nourishes you over time, therefore the hunger can then request additional callbacks from the engine ( neat ;) )

**Comment by smudge202** [comment]
Might want to wait on closing this until I commit game-time branch.  Significant structure change to the engine that I' rather this merged into than vice-versa?
